.circleci/config.yml

```diff
@@ -154,6 +190,55 @@ jobs:
               Write-Host "Forcing build failure due to test failure(s) :'("
               $host.SetShouldExit(1)
             }
+  publish:
```

Comment:
Very similar sequence as on the current scripts. FYI, the current scripts stopped working because of a new redirect in the CircleCI api.

Reply:
Could you give more info on the redirect?

Reply:
Sure, requesting the artifacts of a build via the api now returns a url to where they are hosted instead of immediately providing the artifact payload after using the details in the initial json response. You can see the issue by running `scripts/download-vsix-from-circleci.js` locally.

Reply:
As a separate work item - 
1) Could we add a new script that is doing most of this work here (specifically from 'Packaging extensions' -> 'Create git tag (e.g. v48.1.0) & push changes back to master branch')? 
2) I think we're going to need to do a lot of script clean up so it's not confusing for anyone that may have to do this process manually for whatever reason.

Reply:
@sfsholden agreed, I'll create a work item to follow up on these points.